Mathlib/CategoryTheory/Limits/Shapes/Types.lean

```diff
@@ -929,4 +931,87 @@ lemma pushoutCocone_inl_eq_inr_iff_of_isColimit {c : PushoutCocone f g} (hc : Is
 
 end Pushout
 
-end CategoryTheory.Limits.Types
+end Types
+
+section Multiequalizer
+
+variable (I : MulticospanIndex (Type u))
+
+/-- Given a multiequalizer diagram `I : MulticospanIndex (Type u)` in the category
+of types, this is a type which identifies to the sections of the
+functor `I.multicospan.sections`. -/
```

Comment:
```suggestion
functor `I.multicospan`. -/
```
Also I'd call `I.multicospan` the diagram and `I : MulticospanIndex (Type u)` the index of that diagram.

Reply:
Thanks! I have changed the phrasing.